spring-kafka/src/test/java/org/springframework/kafka/listener/SubBatchPerPartitionTxTests.java

```diff
@@ -201,7 +201,8 @@ public ConcurrentKafkaListenerContainerFactory kafkaListenerContainerFactory() {
 			factory.getContainerProperties().setAckMode(AckMode.BATCH);
 			factory.getContainerProperties().setTransactionManager(tm());
 			factory.setBatchListener(true);
-			factory.getContainerProperties().setSubBatchPerPartition(true);
+			// true by default for a transactional container since 2.5
+//			factory.getContainerProperties().setSubBatchPerPartition(true);
```

Comment:
Probably we don't need this code neither comment about `2.5`

Reply:
This was more for your benefit during review; will remove.